cmd/internal/cli/singularity.go

```diff
@@ -188,6 +189,30 @@ func setSylogColor() {
 	}
 }
 
+// getConfDir queries ConfigDirForUsername() and stops execution if there is an
+// error
+func getConfDir(usr string) string {
+	confDir, err := syfs.ConfigDirForUsername(usr)
+	if err != nil {
+		sylog.Fatalf("Error attempting to determine user's config directory: %s\n", err)
+	}
+	return confDir
+}
+
+// createConfDir tries to create the user's configuration directory and handles
+// messages and/or errors
+func createConfDir(d string) {
+	if err := fs.Mkdir(d, os.ModePerm); err != nil {
+		if os.IsExist(err) {
+			sylog.Debugf("%s already exits. Not creating.", d)
+		} else {
+			sylog.Fatalf("Error attempting to create %s: %s", d, err)
+		}
+	} else {
+		sylog.Debugf("Created %s.", d)
```

Comment:
I don't think there should be a period at the end of this, because it will now print out something looking like this which may be misleading:

`Created /home/mibauer/.singularity.`